screens/shoppingList

```diff
@@ -0,0 +1,286 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image } from 'react-native';
+import FontLoad from './components/FontLoad';
+
+export default class ShoppingList extends React.Component {
+
+
+  render() {
+    return (
+      this.state.fontLoaded ? (
+        <View style={styles.container}>
+           <View style={styles.rectangleOrange} />
+           <View style={styles.rectangleTopLightBlue} />
+           <View style={styles.rectangleMid} />
+           <View style={styles.rectangleBottomLightBlue} />
```

Comment:
Again make sure that `<Text>` components are inside the `<View>` component

Look at how [this](https://github.com/jennyengelman/PongMates/blob/master/screens/SelectionPage.js) file does it